packages/api/src/types.ts

```diff
@@ -57,6 +57,14 @@ export interface IGraphQLRequest {
   variables?: IGraphQLVariables;
 }
 
+export interface IWebhookRequest {
+  method: string;
+  path?: string;
```

Comment:
@sebscholl I implemented `path` option instead of `params` because:
1. We need to make `functionsList` query to understand what path should builded from `params`.
2. There is case when user can't get `functionsList` result because of permissions.

Reply:
Usage example:

```
    const response = await api.invoke(
      'webhookWithParams',
      {
        method: 'POST',
        data: {
          test: 'some value',
        },
        path: 'firstParam/path/secondParam/test',
      },
      {
        headers: {
          auth: 'some auth token',
        },
      },
    );
```

Reply:
Okay, I think that is fine.